### PR TITLE
Add docs for advanced commands

### PR DESCRIPTION
**Pull request by cldwalker**
Following up on my questions in discord :) Also documented block properties invocation and instructions for contributing

### CODE REVIEW

File: pages/advanced_commands.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: Advanced commands
+---
+
+## Advanced commands start with `<` and when selected expand a BEGIN...END section inside a block. There can be multiple BEGIN...END sections in a block.
+## List of advanced commands:
+### `<Quote` - Displays text as quoted by indenting text and giving it left and right margins.
+### `<Src` - Syntax highlights code text using CodeMirror. Passing an argument to the block e.g. `#+BEGIN_SRC clojure` sets the language. Default language is javascript.
+### `<Query` - Defines a block for [[Queries]]. Block renders query results when not being edited.
+### `<Latex export` - TODO
```
Comment (cldwalker) [inline pages/advanced_commands.md:10]: I left TODOs for the ones I was unsure about

File: pages/advanced_commands.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+#### `<Note`
+#### `<Tip`
+#### `<Important`
+#### `<Caution`
+#### `<Warning`
+### `<Example` - Displays text as an example with white space preservation and a different colored background.
+### `<Export` - TODO
+### `<Verse` - Displays text that are for poetry by preserving newlines.
+### `<Ascii` - TODO
+### `<Center` - TODO
```
Comment (cldwalker) [inline pages/advanced_commands.md:22]: I could guess what this one does but it seems buggy. Currently it completes to `<Export`

File: pages/advanced_commands.md (hunk 1 above)
Comment (tiensonqin) [reply pages/advanced_commands.md:10]: Cool, the latex export will be something like:
```
#+BEGIN_EXPORT latex
E = mc2 
#+END_EXPORT
```
It'll be rendered by using Katex.

File: pages/advanced_commands.md (hunk 2 above)
Comment (tiensonqin) [reply pages/advanced_commands.md:22]: Yes, those commands that marked by `TODO` are not implemented yet, I'll remove those commands for the next release.

**Review by tiensonqin — APPROVED** [review]

**Comment by tiensonqin** [comment]
@cldwalker Thank you very much for the help!

File: pages/advanced_commands.md (hunk 2 above)
Comment (cldwalker) [reply pages/advanced_commands.md:22]: Good to know

**Pull request merged**